chef/cookbooks/crowbar/templates/default/crowbar-router-migration.sh.erb

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# This script migrates all neutron routers to the node where the script is
+# executed. As part of the upgrade, the founder node disbanded all routers
+# to other nodes. After the founder is upgraded, all routers have to come
+# back.
+
+LOGFILE=/var/log/crowbar/node-upgrade.log
+UPGRADEDIR=/var/lib/crowbar/upgrade
+mkdir -p "`dirname "$LOGFILE"`"
+exec >>"$LOGFILE" 2>&1
+
+echo "Executing $BASH_SOURCE"
+
+set -x
+
+mkdir -p $UPGRADEDIR
+
+if [[ -f $UPGRADEDIR/router-migration-ok ]] ; then
```

Comment:
This needs to be full name of the file + "-ok" suffix, see https://github.com/crowbar/crowbar-core/blob/master/crowbar_framework/app/models/node_object.rb#L1089